examples/small-study/src/studySetup.js

```diff
@@ -42,9 +42,7 @@ const baseStudySetup = {
       ],
     },
     ineligible: {
-      baseUrls: [
-        "https://qsurvey.mozilla.com/s3/Shield-Study-Example-Survey/?reason=ineligible",
-      ],
+      baseUrls: [],
```

Comment:
We disagree here.  Sometimes it has been, to show a page saying "By the way, you actually don't match this", for opt-in studies.  I will make a note in docs though.

Reply:
Very true. I got blinded by mostly doing opt-out. Thanks for catching this!